.github/workflows/formatting-check.yml

```diff
@@ -6,19 +6,19 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
 
-    - name: Set up JDK
-      uses: actions/setup-java@v1.4.3
-      with:
-        java-version: "14"
+      - name: Set up JDK
+        uses: actions/setup-java@v1.4.3
+        with:
+          java-version: "14"
```

Comment:
As I'm seeing this, 14 is EOL by now. We should either be using 16 (latest one, till tomorrow, when 17 is out ;), or 11, the previous LTS. I suppose there will be issues with 16 due to the changes there in regards to reflective access. Can you capture this in a Jira issue?

Reply:
@gunnarmorling Logged https://issues.redhat.com/browse/DBZ-3993 for upgrading the java version.